src/hotspot/share/gc/z/z_globals.hpp

```diff
@@ -83,6 +83,10 @@
           "(can be any even integer between 4M and 32M)")                   \
           range(4*M, 32*M)                                                  \
                                                                             \
+  experimental(uint, ZUnloadClassesFrequency, 100,                          \
+          "Unload the classes every Nth ZGC cycle."                         \
+          "Set to zero to disable class unloading.")                        \
```

Comment:
Add constraint 0 - max_int?

Reply:
`java -XX:+UseZGC -XX:ZUnloadClassesFrequency=-1 -Xmx10g Xlog:gc*:gc.log:time`

Improperly specified VM option 'ZUnloadClassesFrequency=-1'
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.

no need to check if it is a negative integer.